cfpb/consumerfinance.gov
# Info-unit: Add aria-label from Wagtail to links in template

https://github.com/cfpb/consumerfinance.gov/pull/6497 added an aria-label to links edited in Wagtail, but neglected to add the aria-label to the info-unit template, so the entered aria-labels don't show up in that case.

## Changes

- Add aria-label to info-unit template


## How to test this PR

1. Edit a page with an info-unit and add an aria-label to a link and see it shows up in the published page's info-unit link.
